client/src/components/BuildProfile/index.js

```diff
@@ -57,26 +72,67 @@ const styles = {
   },
   description: {
     width: '40%',
-    height: '100%',
+    height: '80%',
     backgroundColor: '#e9ecef',
     borderRadius: '15px',
     display: 'flex',
     flexDirection: 'column',
     padding: '2%',
-    marginBottom: '2%'
+    marginBottom: '2%',
   },
   edit: {
     display: 'flex',
     alignItems: 'center',
     justifyContent: 'center',
     marginBottom: '5%',
 
-    '&:hover': {
-      cursor: 'pointer'
-    }
   }
+  
+  
+}
+
+// Responsive Code
+//It's still buggy with buttons not changing there positions
+
+
+const viewportContext = React.createContext({});
+
+const ViewportProvider = ({ children }) => {
+  // This is the exact same logic that we previously had in our hook
+
+  const [width, setWidth] = React.useState(window.innerWidth);
+  const [height, setHeight] = React.useState(window.innerHeight);
+
+  const handleWindowResize = () => {
+    setWidth(window.innerWidth);
+    setHeight(window.innerHeight);
+  }
```

Comment:
Nice job with the `useContext` hook here, but typically when making things responsive, we want to avoid manually grabbing the window dimensions.

We can instead use a pure CSS solution, assigning width and hight based on a percentage of a parent container, or a percentage of the user's viewport. If we need to, we can use media queries to assign specific CSS rules based on device dimensions.

Instead of setting the width and height to that of the window, try setting them to a percentage. We can go over that together if you want.